datarobot_batch_scoring/batch_scoring.py

```diff
@@ -647,7 +645,7 @@ def exit_fast(*_):
                         else:
                             ui.info("        {}".format(msg, suffix))
 
-        ui.info('==== Total stats ===='.format(bucket))
+        ui.info('==== Total stats ====')
```

Comment:
In the future, please separate out things that are functionally relevant to your changes from cleanup